src/manifest/ManifestApi.ts

```diff
@@ -34,16 +37,16 @@ export class ManifestApi {
    *
    * @see https://github.com/getoutreach/extensibility-sdk/blob/master/docs/manifest.md#scopes
    * @type {Scopes[]}
-   * @memberof Api
+   * @memberof ManifestApi
    */
   scopes: Scopes[];
 
   /**
    * Address of the endpoint, which will return support refresh token flow.
    *
    * @type {string}
-   * @memberof Api
-   * @deprecated Usage of token endpoint is deprecated and it will be removed in upcoming releases.
+   * @memberof ManifestApi
+   * @deprecated Usage of token endpoint is deprecated and will be removed in upcoming releases.
    */
   token: string;
```

Comment:
Shouldn't we make it also options when deprecating? That applies to all of the deprecated props here.

Reply:
I'm trying it right now and it brings additional complications to validation. Also, some of those fields are used in SDK methods. I'd rather leave them as they are now and remove them completely later. On our side we can take care about filling all of them with values.

Reply:
Ok